libraries/AP_NavEKF2/AP_NavEKF2_core.h

```diff
@@ -1212,4 +1249,10 @@ class NavEKF2_core : public NavEKF_core_common
 
     // vehicle specific initial gyro bias uncertainty
     float InitialGyroBiasUncertainty(void) const;
+
+    // The following declarations are used to control when the main navigation filter resets it's yaw to the estimate provided by the GSF
+	uint64_t EKFGSF_yaw_reset_ms;	        // timestamp of last emergency yaw reset (uSec)
```

Comment:
these should be uint32_t as initialised from a uint32_t

Reply:
will fix